drivers/wifi/eswifi/eswifi_offload.c

```diff
@@ -618,6 +663,68 @@ static int eswifi_off_disable_bypass(struct eswifi_dev *eswifi)
 	return err;
 }
 
+void eswifi_offload_async_msg(struct eswifi_dev *eswifi, char *msg, size_t len)
+{
+	char msg_tcp_accept[] = "[TCP SVR] Accepted ";
+
+	if (!strncmp(msg, msg_tcp_accept, strlen(msg_tcp_accept))) {
+		struct eswifi_off_socket *socket = NULL;
+		struct in_addr *sin_addr;
+		uint8_t ip[4];
+		uint16_t port = 0;
+		char *str;
+		int i = 0;
+
+		/* extract client ip/port e.g. 192.168.1.1:8080 */
```

Comment:
Perhaps you could you use `net_ipaddr_parse()` function as that is able parse that kind of strings.